index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header class="mobile-header">
+        <p>Diego Hernández</p>
+        <button type="button" class="menu"> </button>
+   
+    </header>
+
+
+    <div class="content"> 
+        <img src="./img1.png" alt="background"> 
+        <a> <img src="./smedia.png" alt="dsds"> </a>
+        <h1>Hey There. I'm Diego</h1> 
+        <h3>I’am  web a developer.</h3> 
+        <h4>I would like you to see my some of my best works, I am able to build and design high quality websites. </h4>    
+    </div>
```

Comment:
- [ ] According to your requirements, for this section please use semantic tags as `section` instead of a plain `div`